core/rawdb/freezer_table.go

```diff
@@ -333,16 +339,20 @@ func (t *freezerTable) preopen() (err error) {
 	return err
 }
 
-// truncate discards any recent data above the provided threshold number.
-func (t *freezerTable) truncate(items uint64) error {
+// truncateHead discards any recent data above the provided threshold number.
```

Comment:
Is it above, or from that threshold above?
As I understand, for example, if we have 0 1 2 3 4..., and items = 2, so we're gonna truncate 2, 3, 4,..., I mean including 2, right?

Reply:
threshold here means items input , yes that correct, should keep from 0 to n-1 means 0 and 1